ecommerce/Model/Order.php

```diff
@@ -13,4 +13,12 @@ class Order extends Model
         'total',
         'status'
     ];
+
+    public function orderDetails(){
+        return $this->hasMany(OrderDetail::class);
+    }
+
+    public function users(){
```

Comment:
sửa hết những phần tương tự
```suggestion
    public function user()
    {
```